Resources/public/templates/rawcontent.hbt

```diff
@@ -1,4 +1,5 @@
 <h2 class="ez-raw-content-title"><a href="#">Content</a></h2>
+<div class="ez-language-switcher"></div>
```

Comment:
There's also an inconsistency here, the language switcher is a sub view of the LocationView but its container is added to the RawContentView template. It's either a sub view of the LocationView and its container is added to the LocationView template or a sub view of the RawContentView and it's ok to add the container in the rawcontent.hbt template.